ONSdigital/eq-survey-runner
# Eq 1 Submit Response

**What**
This pull request links the navigation component, login component and questionnaire manager together. It allows the user to start a survey, submit some data and then be presented with the thank you page.

**How to test**
1. Check out this branch
2. Run the run_app.sh script (this should launch the application in chrome).
3. Initial the request should go to /session?token=<jwe>
4. The request should be immediately routed to the cover page
5. On the cover page click start
6. The MCI survey page should be displayed
7. Enter some data and click submit
8. The thank you page should be presented.

**Who can review**
Anyone apart from @warrenbailey and @weapdiv-david
